.jenkins/actions/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+envloc=$1
+RUN_CMD_FILE=$2
+
+. ${envloc}/env/machineEnvironment.sh
+slurmscript="${envloc}/env/submit.${host}.slurm"
+out="gt4py_tests_${BUILD_ID}.out"
+if [ "${useslurm}" = true ] ; then
+    # setup SLURM job
+    /bin/sed -i 's|<NAME>|jenkins-gt4py-tests|g' ${slurmscript}
```

Comment:
Do not hard-code path to `sed`.

Reply:
This was done in the example I built from -- you think we should just say 'sed'?

Reply:
Yep, I think that's the right way to do it.